include/alpaka/mem/buf/BufUniformCudaHipRt.hpp

```diff
@@ -78,7 +78,7 @@ namespace alpaka
                 "The dimensionality of TExtent and the dimensionality of the TDim template parameter have to be "
                 "identical!");
             static_assert(
-                std::is_same_v<TIdx, Idx<TExtent>>,
+                std::is_same_v<TIdx, alpaka::Idx<TExtent>>,
```

Comment:
This should go in the other PR ?

Reply:
you are right. sorry. i removed that change with an extra commit, that can be squashed.